sdk/communication/AzureCommunicationChat/Tests/ChatClientUnitTests.swift

```diff
@@ -39,9 +38,9 @@ class ChatClientUnitTests: XCTestCase {
     private let topic = "test topic"
 
     override func setUpWithError() throws {
-        let settings = TestSettings.loadFromPlist()
-        let endpoint = settings?.endpoint ?? "https://endpoint"
-        let token = settings?.token ?? generateFakeToken()
+        let settings = TestSettings()
+        let endpoint = settings.endpoint
+        let token = settings.token
         let credential = try CommunicationTokenCredential(token: token)
         let options = AzureCommunicationChatClientOptions()
```

Comment:
This appears in `setupWithError()` for just about every Chat test class. Would this be worth refactoring? If so, I'm not sure if that would be something you'd want to do or leave for chat.

Reply:
I'd leave it for the chat folks, but yeah, they could refactor it.